spanner/row_test.go

```diff
@@ -2087,8 +2088,8 @@ func TestSelectAll(t *testing.T) {
 				},
 			},
 			want: &[]testStruct{
-				{Col1: 1, Col2: 1.1, Col3: "value", Col4: tm},
-				{Col1: 2, Col2: 2.2, Col3: "value2", Col4: tm.Add(24 * time.Hour)},
+				{Col1: 1, COL2: 1.1, Col3: "value", Col4: tm},
```

Comment:
Same here also: Should this not be kept as `Col2`?

Reply:
No, this should be `COL2`, because testStruct is user defined struct and we are asserting that though mock response contained `col2` it was successfully decoded in testStruct